### ISSUE TITLE
modules/diskq/tests/test_diskq fails on 32 bit platforms

### ISSUE THREAD

**Issue by lbudai** [balabit/syslog-ng#1545]
# syslog-ng

## Version of syslog-ng

```
 * 3.10
```

## Platform

32 bit linux

# Issue

FAIL: ASSERTION FAILED: one_msg_size 384: line: 465; actual=FALSE, expected=TRUE

# the code that cause the problem:
```
gboolean
is_valid_msg_size(guint32 one_msg_size)
{
    /* 460 as of writing */
    return (one_msg_size < 500 && one_msg_size > 400);
}
```

**Comment by mochrul** [comment]
Debian packaging fails because of this issue.

**Comment by furiel** [comment]
There was a recent statistics feature that introduced memory usage for queues. As part of it, we needed to measure the memory usage of a single message. Though it is hard to verify such size. So once I got some reasonable value (on x86-64), I added some guard values to track this value in the future, or in case we can out regressions related  to this. But these values just came from a crystal ball, and did not take other architectures into consideration. Not sure if we can give reasonable numbers for all architectures.

For now I will create a patch to set lower bound to 300. In case we run into more issues with this, then I am afraid we need to disable this check, i.e just asserting nonzero size.
